app/controllers/subscriptions_controller.rb

```diff
@@ -75,6 +75,17 @@ def destroy
 
   private
 
+  def track_origin_and_trigger_event
+    return unless origin_param&.start_with?(%r{/\w})
```

Comment:
What does the forward slash `/` do?

Is the purpose here to only track the origin if there is an origin?

Reply:
yes I believe so, I just moved this logic, did not change it